samples/nrf9160/nrf_cloud_mqtt_multi_service/overlay_full_modem_fota.conf

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Full Modem FOTA
+CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE=y
+CONFIG_NRF_MODEM_LIB_SYS_INIT=n
+CONFIG_SPI=y
+CONFIG_SPI_NOR=y
+CONFIG_CONNECTION_THREAD_STACK_SIZE=4096
```

Comment:
I think enabling this by default rather than having it as an overlay would be fine

Reply:
Not all boards support full modem FOTA, because not all boards have the required external flash.  Or are you just referring to line 11?

Reply:
i'd rather leave this as optional... full modem fota enables a lot of other libraries, see kconfig for NRF_CLOUD_FOTA_FULL_MODEM_UPDATE